test/Cardano/BM/Test/Trace.lhs

```diff
@@ -490,30 +489,32 @@ setVar_ = do
 unit_name_filtering :: Assertion
 unit_name_filtering = do
     let contextName = "test.sub.1"
-    let logobject = LogValue "sum" (PureD 42.0)
+    let loname = "sum"
+    -- let logobject = LogValue loname (PureD 42.0)
```

Comment:
Is this comment needed?

Reply:
actually not, just to indicate where that name would come from.